rails_event_store_active_record/lib/rails_event_store_active_record/mappers/yaml.rb

```diff
@@ -0,0 +1,31 @@
+module RailsEventStoreActiveRecord
+  module Mappers
+    class YAML
+      def initialize(serializer: nil)
+        @serializer = serializer || ::YAML
+      end
+
+      def build_event_payload(event)
+        {
+          id:         event.event_id,
+          data:       serializer.dump(event.data),
+          metadata:   serializer.dump(event.metadata),
+          event_type: event.class,
+        }
+      end
+
+      def build_event_entity(record)
```

Comment:
`build_event_entity` - an event is not an entity

Reply:
name and responsibility its extracted from repository

Reply:
I'm open for name proposal